app/assets/javascripts/common_chart_options.js

```diff
@@ -92,6 +92,10 @@ function barColumnLine(d, c, chartIndex, seriesData, chartType, noZoom) {
   // LINE charts
   if (chartType == 'line') {
     if (y2AxisLabel !== undefined && y2AxisLabel.length) {
+
+      var axisTitle;
```

Comment:
whoops, bit naughty that I'd left those out at some point, thanks for popping them back in!